native/src/main/java/io/ballerina/stdlib/io/nativeimpl/RecordChannelUtils.java

```diff
@@ -186,6 +186,9 @@ public static Object getAllRecords(BObject channel, int skipHeaders, BTypedesc t
                 }
                 Object[] out = outList.toArray();
                 return ValueCreator.createArrayValue(out, TypeCreator.createArrayType(describingType));
+            } else {
+                return IOUtils.createError(String.format("Only 'string[]' and record{} are supported, " +
```

Comment:
```suggestion
                return IOUtils.createError(String.format("Only 'string[]' and 'record {}' types are supported, " +
```

Reply:
updated in [ec66f19](https://github.com/ballerina-platform/module-ballerina-io/pull/898/commits/ec66f19f98d5e5c2a8590e2da49696154ce9a8f4)